helloworld/translate/models.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2021 Pants project contributors.
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from django.db import models
+
+from helloworld.greet.models import Greeting
+
+
+class Translation(models.Model):
+    class Meta:
+        constraints = [
+            models.UniqueConstraint(fields=["greeting", "lang"], name="greeting_lang")
+        ]
+
+    greeting = models.ForeignKey(Greeting, on_delete=models.CASCADE)
```

Comment:
Nitpick: I suspect people will notice the hard code dependency between two services (is this also non-brokered database sharing?), which is generally frowned upon as a microservices architectural principle. Not sure this is worth tying up in knots over, but I think it's noticeable